packages/framework/block-std/package.json

```diff
@@ -45,7 +46,8 @@
       ".": {
         "module": "./dist/index.js",
```

Comment:
We should align the configs for each entry points.

Reply:
I checked the docs and seems that if the config between `module` and `import` are the same, we don't have to repeat them. So I suggest making the config simpler.